components/filter/filter.js

```diff
@@ -120,12 +121,15 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 				white-space: nowrap;
 			}
 
-			.d2l-filter-dimension-set-value-text {
+			.d2l-filter-dimension-set-value {
 				color: var(--d2l-color-ferrite);
```

Comment:
What colour are the counts supposed to be?  The design looks like they're supposed to be lighter than `ferrite`, but you may have a newer copy than the one linked in the story.  The disabled case also looks a bit odd, wondering if we should at least make it lighter in that case

Reply:
The `d2l-body-small` styles includes setting color to `tungsten`.

Reply:
Ah gotcha!  But yeah I wonder if when disabled, we match the text

Reply:
@ksanctis , can you follow up with Jeff to find out what he wants? I suspect we want to match the disabled color when it's disabled as Stacey mentioned.

Reply:
I will confirm with Jeff and make changes accordingly.

Reply:
Got confirmation from Jeff, will make the colour for the count the same as the text when the filter is disabled.